packages/runtime/container-runtime/src/opLifecycle/remoteMessageProcessor.ts

```diff
@@ -49,65 +49,49 @@ export class RemoteMessageProcessor {
 	 * a singleton array [remoteMessageCopy] is returned
 	 */
 	public process(
-		remoteMessageCopy: ISequencedDocumentMessage,
+		remoteMessage: ISequencedDocumentMessage,
```

Comment:
It was intentional to call this `Copy`, to indicate it can be modified without affecting other parts of the system.  See the doc comment above.